cmake/script/CoverageInclude.cmake.in

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+if("@CMAKE_CXX_COMPILER_ID@" STREQUAL "Clang")
+  find_program(LLVM_COV_EXECUTABLE llvm-cov REQUIRED)
+  set(COV_TOOL "${LLVM_COV_EXECUTABLE} gcov")
+else()
+  find_program(GCOV_EXECUTABLE gcov REQUIRED)
+  set(COV_TOOL "${GCOV_EXECUTABLE}")
+endif()
+
+# COV_TOOL is used to replace a placeholder.
+configure_file(
+  cmake/cov_tool_wrapper.sh.in ${CMAKE_CURRENT_LIST_DIR}/cov_tool_wrapper.sh
+  FILE_PERMISSIONS OWNER_READ OWNER_EXECUTE
+                   GROUP_READ GROUP_EXECUTE
+                   WORLD_READ
+  @ONLY
+)
+
+set(LCOV_OPTS --ignore mismatch)
```

Comment:
I don't think this should be hardcoded here (it's not needed with lcov 1.x, and isn't the only ignore that might be required with 2.x)?

Reply:
Thanks! [Deleted](https://github.com/hebasto/bitcoin/pull/191#issuecomment-2139141905).